setup.py

```diff
@@ -59,10 +59,6 @@ def read_file(filename):
       author_email='alexis@notmyidea.org',
       url='https://github.com/spiral-project/ihatemoney',
       packages=find_packages(),
-      package_data={'': ['*.rst', '*.py', '*.yaml', '*.po', '*.mo', '*.html',
```

Comment:
if I understand properly, if we add it in the `recursive-include` we don't have to have the `package_data` and `include_package_data` entries?